src/libraries/System.Memory.Data/src/System/BinaryData.cs

```diff
@@ -201,38 +302,50 @@ private static async Task<BinaryData> FromStreamAsync(Stream stream, bool async,
                 }
                 else
                 {
+#pragma warning disable CA1849 // Call async methods when in an async method
                     stream.CopyTo(memoryStream, bufferSize);
+#pragma warning restore CA1849 // Call async methods when in an async method
```

Comment:
We don't have CA1849 enabled as a warning or error.  This suppression should not be necessary.  Please revert.